proto/sf/substreams/rpc/v2/service.proto

```diff
@@ -69,6 +69,12 @@ message Request {
   // Progress_messages_interval_ms is the interval between progress messages, in milliseconds (minimum: 500, default: start at 500ms and ramp up to 5000ms within 1min)
   uint64 progress_messages_interval_ms = 14;
 
+  // If true, partial blocks are also sent on the stream
+  bool include_partial_blocks = 15;
+
+  // If true, only partial blocks are sent, no 'block-scoped-data' or cursor.
+  // This value supersedes include_partial_blocks
+  bool partial_blocks_only = 16;
```

Comment:
Let's add more documentation to the Protobuf Stream + Fields:
- Sequence of messages with partial blocks
- Relation about the two fields, for example is it possible to have `partial_blocks_only: true` and `include_partial_blocks:false`? Does one imply the other.
- All important information about the "contract" of partial blocks.